plugins/sentry/girder_sentry/settings.py

```diff
@@ -3,15 +3,22 @@
 
 
 class PluginSettings(object):
-    DSN = 'sentry.dsn'
+    _FRONTEND_DSN = 'sentry.frontend_dsn'
```

Comment:
We want these to be public symbols, as all other plugin settings are.

Sorry that you have to fight with the public symbols test. 🙁